src/repository.ts

```diff
@@ -21,6 +23,9 @@ import { upload } from './codeSigning/upload';
 import { Package, VersionValidation } from './package';
 import { Registry } from './registry';
 
+const conventionalCommitsParser = require('conventional-commits-parser');
```

Comment:
Why do these need to be brought in as require? Can't you still do `import * as conventionalCommitsParser from 'conventional-commits-parser'`?